<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCorrectTemplateDlgClass</class>
 <widget class="QDialog" name="CreateCorrectTemplateDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateCorrectTemplateDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_offset">
      <attribute name="title">
       <string>Create Offset</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_OffsetGenerationAbort">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_OffsetGenerationStart">
         <property name="text">
          <string>Start create offset template file</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_gain">
      <attribute name="title">
       <string>Create Gain</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="4">
        <widget class="QLineEdit" name="lineEdit_GainExceptedValue">
         <property name="text">
          <string>12000</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_GainTargetCurrent">
         <property name="text">
          <string>Target Current:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="label_GainCenterValue">
         <property name="text">
          <string>CurrrentValue</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3" colspan="2">
        <widget class="QPushButton" name="pushButton_GainSelectAll">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QLineEdit" name="lineEdit_GainCenterValue">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="3" colspan="2">
        <widget class="QPushButton" name="pushButton_GainGeneration">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="7" colspan="2">
        <widget class="QGraphicsView" name="graphicsView_GainExceptedValue"/>
       </item>
       <item row="4" column="3" colspan="2">
        <widget class="QPushButton" name="pushButton_GainStartAcq">
         <property name="text">
          <string>Start Acq</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QSpinBox" name="spinBox_GainTargetCurrent"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_GainAbort">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_GainExceptedValue">
         <property name="text">
          <string>Exepcted Value:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_GainSuggestedKV">
         <property name="text">
          <string>Suggested KV:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_GainInit">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="lineEdit_GainSuggestedKV">
         <property name="text">
          <string>70</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_defect">
      <attribute name="title">
       <string>Create Defect</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="7" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>CurrrentValue</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="2">
        <widget class="QPushButton" name="pushButton_15">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Target Current:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QSpinBox" name="spinBox_3"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item row="1" column="0" rowspan="10" colspan="2">
        <widget class="QGraphicsView" name="graphicsView_3"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Stage:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Suggested KV:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_14">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string>Start Acq</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2" colspan="2">
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2" colspan="2">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Expected Value:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
